validation-test/stdlib/String.swift

```diff
@@ -408,7 +408,7 @@ StringTests.test("appendToSubstring") {
           sliceEnd < sliceStart {
           continue
         }
-        var s0 = String(repeating: UnicodeScalar("x"), count: initialSize)
+        var s0 = String(repeating: String(UnicodeScalar("x")), count: initialSize)
```

Comment:
Sure. I thought it could be interesting to show how one could "migrate" from UnicodeScalar form of the constructor to a String-based one. But other than that, yes, it is cleaner to use a shorter form that you mentioned.